osu.Framework.Android/AndroidGameView.cs

```diff
@@ -220,7 +218,7 @@ private void updateSafeArea()
 
             int[] location = new int[2];
             GetLocationOnScreen(location);
-            var viewArea = new RectangleI(location[0], location[1], Width, Height);
+            var viewArea = new RectangleI(location[0], location[1], (this as View).Width, (this as View).Height);
```

Comment:
Don't know if it's just me but it seems really weird to be using soft casts that can't fail. Would prefer direct casts in this instance.

```suggestion
            var viewArea = new RectangleI(location[0], location[1], ((View)this).Width, ((View)this).Height);
```